frontend/src/index.css

```
@@ -37,14 +37,15 @@
 
 /* General Styles */
 * {
+  font-family: var(--primary-font);
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-weight: var(--font-weight-medium);
```

Combine related styles to enhance readability and maintainability 
```suggestion
  font-family: var(--primary-font);
  font-weight: var(--font-weight-medium);
  box-sizing: border-box;
  margin: 0;
  padding: 0;
```

---

I've combined the styles